HealthMonitoring.Integration.PushClient/HealthMonitorPushClient.cs

```diff
@@ -51,18 +52,29 @@ public HealthMonitorPushClient WithHealthCheck(IHealthChecker healthChecker)
             return this;
         }
 
+        public HealthMonitorPushClient WithBackOffStategy(IBackOffStategy backOffStategy)
+        {
+            _backOffStategy = backOffStategy;
```

Comment:
actually sorry, i thought of wrong client, but nevertheless what else strategies are we going to have except default ?